src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
```

Comment:
Ah, well that's an hour of code crawling well spent then.

The problem now is that the gas spawns in the tile the player is in, not the tile at which the launcher aims. Do I have to define a projectile somewhere?